postprocessing/janus-pp-rec.c

```diff
@@ -117,6 +117,7 @@ Usage: janus-pp-rec [OPTIONS] source.mjr
 #include <jansson.h>
 
 #include "../debug.h"
+#include "../utils.h"
```

Comment:
I'll have to check if this means we need to add `utils.o` as an object when building `janus-pp-rec`. I see the automated Github Actions checks have all succeeded, so this shouldn't be an issue.

Reply:
`utils.c` is already listead as part of `janus_pp_rec_SOURCES` in `Makefile.am` so I think there's no change really.